specification/monitor/resource-manager/microsoft.insights/2016-09-01/serviceDiagnosticsSettings_API.json

```diff
@@ -181,7 +181,7 @@
               "$ref": "#/parameters/ApiVersionParameter"
             },
             {
-                "name": "serviceDiagnosticSettingsResource",
+                "name": "parameters",
```

Comment:
@gucalder May I ask the reason for this change i.e renaming the parameter. Seems like generating new SDK from this swagger would not be backward compatible as parameter name has changed. Can you please provide your thoughts.